templates/partners/kubernetes.html

```diff
@@ -0,0 +1,86 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <section class="p-strip--image is-dark is-deep u-no-padding--top u-no-padding--bottom u-extra-space">
+    <div class="p-strip--suru-background">
+      <div class="row">
+        <div class="col-6">
+          <h1>Charmed Kubernetes partners</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore deleniti eaque repellendus, impedit nobis veniam, pariatur qui quae laboriosam dolores doloremque iusto nesciunt nihil doloribus officia iste modi, repellat consectetur!</p>
+        </div>
+        <div class="col-6 u-align--center u-hide--small">
+          <img src="https://assets.ubuntu.com/v1/0038a560-charmed-kubernetes-partners-cloud.svg" alt="charmed kubernetes image">
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="p-strip is-deep u-extra-space">
+    <div class="row u-sv3">
+      <p class="p-heading--two">Technology and product partners</p>
```

Comment:
This should be a `<h2>` element, at the moment the page heirarchy jumps from the `<h1>` header on line 8, straight to the `<h3>` headers on lines 24, 31 and 41.

![Screenshot 2019-10-07 at 17 03 11](https://user-images.githubusercontent.com/2376968/66328437-8884a680-e924-11e9-89b7-130268b07b7c.png)